modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/EditObjectRelationship.tsx

```diff
@@ -10,13 +10,12 @@ import {
 	openToast,
 	saveAndReload,
 } from '@liferay/object-js-components-web';
+import {ILearnResourceContext} from 'frontend-js-components-web';
```

Comment:
No need to change, but please leave SF changes in a separate commit or squash to [where the changes were originated](https://github.com/liferay-bpm/liferay-portal/pull/4266/commits/b3e991256989d159c339604cd1fd07ed9f3dcd9c#diff-16696f7e91937c8088aa69445080b1104fa54ac7c0673f199c34d17de7cfc1d6). This will reduce the cognitive load for reviewers and keep changes where they really happened. This applies to other changes in this commit.